language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ClientResourceAccessActionNodeContext.java

```diff
@@ -266,4 +287,15 @@ private List<LSCompletionItem> getNamedArgExpressionCompletionItems(BallerinaCom
         return getNamedArgCompletionItems(context, functionSymbol,
                 node.arguments().get().arguments());
     }
+
+    private boolean isGetMethodWithoutParams(ResourceMethodSymbol resourceMethodSymbol) {
+        Optional<String> name = resourceMethodSymbol.getName();
+        FunctionTypeSymbol functionTypeSymbol = resourceMethodSymbol.typeDescriptor();
+        if (name.isPresent() && !"get".equals(name.get())) {
```

Comment:
Shall we use this instead?
```suggestion
        if (name.isPresent() && !name.get().equals("get")) {
```

Reply:
The best practice is to use ` "get".equals(name))`. Is there any reason behind your suggestion?